aristanetworks/ansible-avd
# Feat(eos_designs): Add platform settings for 7368X4

## Change Summary

Add support for platform specific default variable (management interface and reload delay counters for mlag and non-mlag) for 7368X4 series.

## Related Issue(s)

Fixes #1680

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes

Added an entry for platform series 7368X4 in the `roles/eos_designs/defaults/main/main.yml` file reflecting the platform defaulrs:

```
  - platforms: [ '7368X4']
    management_interface: Management0
    reload_delay:
      mlag: 300
      non_mlag: 330
```

* 7368X4 is a modular chassis and its management interface is Management0
* The HW platform timers for 7368X4 are mlag 300ms and non-mlag 300ms. In order to follow the EVPN guide best practices, the non-mlag timer is set by default for this platform to 330ms.
* Test added in molecule to generate the config to verify both settings
* Documentation updated

## How to test

Molecule test added to generate two configurations for platform 7368X4 type to verify both management interface and reload delay counters

## Checklist

### User Checklist


- N/A

### Repository Checklist
